components/datetime/tests/fixtures/tests/components.json

```diff
@@ -36,8 +36,8 @@
                 "fr-u-ca-tbla": "mardi 26 joumada al oula 1441 ère de l’Hégire 08:25:07",
                 "en-u-ca-persian": "Tuesday, Bahman 01, 1398 AP, 08:25:07",
                 "fr-u-ca-persian": "mardi 01 bahman 1398 Anno Persico 08:25:07",
-                "en-u-ca-hebrew": "Tuesday, 24 Tevet 5780 Anno Mundi, 08:25:07",
-                "fr-u-ca-hebrew": "mardi 24 téveth 5780 Anno Mundi 08:25:07",
+                "en-u-ca-hebrew": "Tuesday, 24 Tevet 5780 hebrew, 08:25:07",
```

Comment:
hm, this is broken again

Reply:
it broke once I changed the era to "hebrew", I think I should leave it as "am" which stands for Anno Mundi.

Reply:
No, you need to update the era code map in datagen as well.

The code is supposed to be `"hebrew"`, and we should make all the changes needed to make that work.

Reply:
The CLDR JSON data is not very pretty. It has "Adar I" at "6", "Adar" at "7", and "Elul" at 13. The two solutions are to either map the CLDR keys to our month code keys, which is what Pedro had yesterday, or to depend on the order of the CLDR JSON iteration and create a map in that order (but without parsing the CLDR keys).

Reply:
yes, this is exactly what I had suggested earlier; Pedro's earlier code was a result of my suggestion

Reply:
What are the other problems?